### PR TITLE
Rename 'get' to 'install'

### PR DESCRIPTION
**Pull request by bpinto**
We already use 'install' on the command description: 'Install one or
more packages.'

Also call update using 'u' instead of 'p'.

### CODE REVIEW

**Comment by bucaran** [comment]
We can use both?

**Comment by bpinto** [comment]
Yep, get is still available.

File: pkg/omf/completions/omf.fish [hunk 1]
```diff
@@ -14,11 +14,11 @@ complete --no-files -c omf -d "Oh My Fish"
 
 complete -c omf -n "__omf_opt_is q query"      -a (printf "%s " (set | awk '{ printf $1"\n"; }'))
 complete -c omf -n "__omf_opt_is r rm remove"  -a (printf "%s " (omf_list_local_packages) (omf_list_installed_themes))
-complete -c omf -n "__omf_opt_is g get"        -a (printf "%s " (omf_list_db_packages))
+complete -c omf -n "__omf_opt_is g install"    -a (printf "%s " (omf_list_db_packages))
```
Comment (bucaran) [inline pkg/omf/completions/omf.fish]: Did you forget `i`?

Comment (bpinto) [inline pkg/omf/completions/omf.fish]: I did.

Comment (bucaran) [inline pkg/omf/completions/omf.fish]: Let's merge the other one first?

Comment (bpinto) [inline pkg/omf/completions/omf.fish]: Sure.

**Comment by bucaran** [comment]
LGTM :+1:

**Pull request merged**